faunadb-java/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -2201,6 +2201,24 @@ public static Expr HasIdentity() {
     return Fn.apply("has_identity", Null());
   }
 
+  /**
+   * Creates a new access provider given the params.
+   *
+   * @param params  An object of parameters used to create a new access provider.
+   *  - name: A valid schema name
```

Comment:
We need to update the params here! :)

Reply:
If I remember correctly, `allowed_roles` and `allowed_collections` fields do not exist anymore (thus we should remove them from here).

@marrony & @freels, could you confirm that? :)

Reply:
correct